backends/windows/kernelspace/sink.go

```diff
@@ -104,7 +104,9 @@ func (s *Backend) DeleteEndpoint(namespace, serviceName, key string) {
 }
 func (s *Backend) SetService(svc *localnetv1.Service) {}
```

Comment:
i think we'll need to impl `SetService` and/or `SetEndpoint` , at the very minimum, before doing any meaningful testing